scripts/deploy

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -e
+
+if [ "$TRAVIS_PULL_REQUEST" = "true" ] || [ "$TRAVIS_BRANCH" != "master" ]; then
+  docker buildx build \
+    --progress plain \
+    --platform=linux/amd64,linux/arm/v7 \
+    .
+  exit $?
+fi
+
+echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin &> /dev/null
```

Comment:
How it should source the Docker Password? Travis secrets?
 We should create docker hub user just for be consumed by travis?

Reply:
Something like this? https://docs.travis-ci.com/user/encryption-keys#usage

Reply:
Sorry, I forgot to add the configuration part.
Anyway, it's how you said: we have to create a user in the docker hub to be consumed by travis and we have to set the environment variables from travis repo's settings.

I did the last operation from the UI, not from the CLI as you linked, and it encrypted the variables by default.

Reply:
Sending credentials in private

Reply:
Did you already set the right `DOCKER_USERNAME` or I must replace it?